charms/kfp-api/src/charm.py

```diff
@@ -162,9 +169,9 @@ def _kfp_api_layer(self) -> Layer:
             "checks": {
                 "kfp-api-up": {
                     "override": "replace",
-                    "period": "30s",
-                    "timeout": "20s",
-                    "threshold": 4,
+                    "period": "5m",
```

Comment:
5 minutes feels like a very long time for the health check. I understand that we are fixing low resources GH runners issue, but having such long period for health check is not good for real deployment. With threshold of 3 we will know that kfp-api is down after 15 minutes. Unlsess I am misreading this.